lib/v2/xinhua-daily-telegraph/daily.js

```diff
@@ -0,0 +1,71 @@
+const timezone = require('@/utils/timezone');
+const cheerio = require('cheerio');
+const { parseDate, getElementChildrenInnerText, getHTMLWithCache } = require('./utils');
+
+module.exports = async (ctx) => {
+    const date = timezone(new Date(), 8);
+    const dateFormatted = date.getFullYear().toString() + ('0' + (date.getMonth() + 1).toString()).slice(-2) + date.getDate().toString();
+
+    const baseUrl = `http://mrdx.cn/content/${dateFormatted}`;
+    const link = `${baseUrl}/Page01DK.htm`;
+
+    const $ = cheerio.load(await getHTMLWithCache(link, ctx));
+
+    const passagesList = $('.listdaohang > ul > li a');
+    const passageLinks = [];
+
+    // 当日所有文章Url在<a>的`daoxiang` attribute中
+    // 其站点使用**非标准**的HTML语法
+    // eg: `<a class="foo" daoxiang="foo-bar.htm">Lorem ipsum</a>`
+    for (let i = 0; i < passagesList.length; i++) {
+        passageLinks.push(passagesList[i].attribs.daoxiang);
+    }
+
+    const passages = [];
+    const results = await Promise.all(
+        passageLinks.map(async (passage) => {
+            const link = `${baseUrl}/${passage}`;
+            const response = await getHTMLWithCache(link, ctx);
+            return cheerio.load(response);
+        })
+    );
+
+    for (const $ of results) {
+        // 标题在`h2`,`h3`,`h4`,`h5`中，有多余空格和换行符
+        const title =
```

Comment:
`title` will be trimmed by middleware https://github.com/DIYgod/RSSHub/blob/df521a6f2c4ba91fff781b36e8fe785bb11e7d87/lib/middleware/template.js#L52